src/Tribe/Editor/Warnings.php

```diff
@@ -157,4 +147,79 @@ public function render_notice( $message, $type = 'info', $depends_on = '', $cond
 		<?php
 	}
 
+	/**
+	 * Check whether the recurring warning should be displayed or not.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $post_id The current edited post id.
+	 *
+	 * @return bool
+	 */
+	protected function should_display_recurring_warning_for_tickets( int $post_id ): bool {
+		if ( ! class_exists( 'Tribe__Events__Pro__Main' ) || ! class_exists( TEC::class ) ) {
```

Comment:
Both here should pass the second param to avoid autoloading correct?

Reply:
Thanks, updated now.